code/modules/mob/living/carbon/human/npc/dungeon_pack_npc.dm

```diff
@@ -0,0 +1,768 @@
+/mob/living/carbon/human/species/human/northern/dungeon_base
+	aggressive = TRUE
+	mode = AI_IDLE
+	gender = MALE
+	faction = list("dungeon")
+	ambushable = FALSE
+	dodgetime = 20
+	defprob = 35
+	flee_in_pain = FALSE
+	wander = TRUE
+
+	var/combat_loop_running = FALSE
+	var/next_cast = 0
+
+/mob/living/carbon/human/species/human/northern/dungeon_base/examine(mob/user)
+	to_chat(user, "You don't recognize this person.")
+
+/mob/living/carbon/human/species/human/northern/dungeon_base/Initialize()
+	. = ..()
+	set_species(/datum/species/human/northern)
+	spawn(1)
+		after_creation()
+
+/mob/living/carbon/human/species/human/northern/dungeon_base/after_creation()
+	..()
+	gender = pick(MALE, FEMALE)
+	job = "Wanderer"
+	ADD_TRAIT(src, TRAIT_NOMOOD, TRAIT_GENERIC)
+	ADD_TRAIT(src, TRAIT_NOHUNGER, TRAIT_GENERIC)
+	ADD_TRAIT(src, TRAIT_NOSTAMINA, TRAIT_GENERIC)
+	ADD_TRAIT(src, TRAIT_CRITICAL_RESISTANCE, TRAIT_GENERIC)
+	ADD_TRAIT(src, TRAIT_HEAVYARMOR, TRAIT_GENERIC)
+	ADD_TRAIT(src, TRAIT_ACCURACY_MID_NPC, TRAIT_GENERIC)
+	ADD_TRAIT(src, TRAIT_PARRY_MID_NPC, TRAIT_GENERIC)
+
+
+
+/mob/living/carbon/human/species/human/northern/dungeon_base/Life()
+	. = ..()
+	npc_idle()
+	npc_combat()
```

Comment:
NOOOOOO. Do NOT do this. If you need them to do AI steps *PLEASE PLEASE PLEASE PLEASE PLEASE PLEASE PLEASE* just call handle_ai().